templates/profile_view/profile.html

```diff
@@ -8,26 +8,56 @@
     <link rel="stylesheet" href="{% static 'profile_view/style.css' %}">
 </head>
 <body>
+    <header>
+        <nav>
+            <ul class="operation">
+                <li><a href="#">Explorer</a></li>
+                <li><a href="#">Notification</a></li>
+                <li><a href="/create-section/">+ Add Section</a></li>
```

Comment:
add section shouldn't be here. it will be a button on the course view page because you need a course to already be selected to add a section.

Reply:
Got it. Remove later. Just follow the initial design haha